main.js

```diff
@@ -0,0 +1,19 @@
+var backgroundButtons = document.getElementById("background-buttons");
```

Comment:
I appreciate making sure that the variables are named logically. They do what they say they do. Same with the function changeBackground